codebuild/codebuild_python_tox/code_build_python_tox.tf

```diff
@@ -0,0 +1,48 @@
+resource "aws_codebuild_project" "code_pipeline_python_tox" {
+  name        = "${var.pipeline_name}-python-tox-${var.environment}"
+  description = "Run Python tox"
+
+  service_role = data.aws_iam_role.execution_role.arn
+
+  artifacts {
+    type = "CODEPIPELINE"
+  }
+
+  cache {
+    type  = "LOCAL"
+    modes = ["LOCAL_DOCKER_LAYER_CACHE", "LOCAL_SOURCE_CACHE"]
+  }
+
+  environment {
+    compute_type                = "BUILD_GENERAL1_SMALL"
+    image                       = var.codebuild_image
+    type                        = "LINUX_CONTAINER"
+    image_pull_credentials_type = "SERVICE_ROLE"
+    privileged_mode             = false
+
+    environment_variable {
+      name  = "AWS_ACCOUNT_ID"
+      value = var.deployment_account_id
+    }
+
+    environment_variable {
+      name  = "PYTHON_SOURCE_DIRECTORY"
+      value = ""
```

Comment:
how come you're passing this through as an empty string?

Reply:
ahh I wasn't sure about this, I figured you would only know the Python version and the source code directory when you're using the module e.g.

```
module "keycloak_codepipeline_module" {
  source                  = "../../modules/codepipeline"
  environment             = "test"
  account_id              = "103495720024"
  python_source_directory = "3.7.5"
  python_version          = "config_builder"
}
```

Reply:
ah I see - in that case should we pass them through as variables, ie `value = var.python_source_directory`

Reply:
yep that makes more sense, thanks will get that updated.